addons/map/XEH_postInitClient.sqf

```diff
@@ -19,4 +18,10 @@ call FUNC(determineZoom);
     ((findDisplay 12) displayCtrl 51) ctrlAddEventHandler ["Draw", {[] call FUNC(updateMapEffects);}];
 };
 
-ADDON = true;
+["SettingsInitialized", {
+    // Start Blue Force Tracking if Enabled
+    if (GVAR(BFT_Enabled)) then {
```

Comment:
Probably want a `if !(hasInterface) exitWith {};` here so we don't start the PFEH on server/headless